roles/pulp_common/defaults/main.yml

```diff
@@ -15,10 +15,9 @@ pulp_user: pulp
 pulp_user_id:
 pulp_group: pulp
 pulp_group_id:
-pulp_user_home: '/var/lib/pulp'
```

Comment:
`pulp_user_home` should stay that way. It's the actual system defined home directory of that service user.